src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -174,6 +171,24 @@ const TRANSFORM_AGGREGATES = {
   error_rate: '',
 } as const;
 
+function transformAggregate(fieldName: string): string {
+  // test if a field name is a percentile field name. for example: p50
+  if (/^p\d+$/.test(fieldName)) {
+    return 'transaction.duration';
+  }
+
+  return TRANSFORM_AGGREGATES[fieldName] || '';
+}
+
+function isTransformAggregate(fieldName: string): boolean {
```

Comment:
Instead of duplicating this logic you could use `tranformAggregate(fieldname) !== ''`